karrot/subscriptions/receivers.py

```diff
@@ -39,14 +41,20 @@
     get_feedback_possible
 from karrot.subscriptions import tasks
 from karrot.subscriptions.models import ChannelSubscription
-from karrot.subscriptions.utils import send_in_channel, MockRequest
+from karrot.subscriptions.utils import send_in_channel, MockRequest, receiver_transaction_task
 from karrot.userauth.serializers import AuthUserSerializer
 from karrot.users.serializers import UserSerializer
-from karrot.utils.misc import on_transaction_commit
 
+pre_delete_with_cloned_instance = Signal()
 
-@receiver(post_save, sender=ConversationMessage)
-@on_transaction_commit
+
+@receiver(pre_delete)
+def clone_instance_and_forward(signal, sender, instance, **kwargs):
```

Comment:
This provides `id` fields to signal receivers that used `pre_delete` before - during object deletion, Django sets the `id` field to `None`, making it unavailable to `pre_delete` receivers that run `on_commit`.

Reply:
Seems this throws some errors in other parts of the code :( Well, maybe I need another solution...